run_dir/design/deliveries.html

```diff
@@ -23,44 +23,36 @@
 
 <div id="page_content" class="deliveries-page">
   <div class="row">
-    <div>
-      <h1>Ongoing <span id="page_title">Deliveries</span></h1>
-      <h3>Total number of:</h3>
-      <h5>projects: {{ number_of_projects }}</h5>
-      <h5>flowcells: {{ number_of_flowcells }}</h5>
-      <h5>lanes: {{ number_of_lanes }}</h5>
-      <h5>samples: {{ number_of_samples }}</h5>
       <div id="ongoing_deliveries">
-
-        <div class="delivery-filters" style="display:none;">
+        <h1>Ongoing <span id="page_title">Deliveries</span></h1>
+        <h3>Total number of:</h3>
```

Comment:
I expected there to be a number after the `:` somehow. I think the page is probably clearer if you delete this line.